resources/resource_transformers/js/build_test.go

```diff
@@ -42,6 +42,7 @@ func TestToInternalOptions(t *testing.T) {
 		JSXFactory:  "v3",
 		JSXFragment: "v4",
 		Externals:   []string{"react"},
+		Defines:     map[string]interface{}{"process.env.NODE_ENV": "production"},
```

Comment:
It may be too late now because this is already merged and released, but wouldn't it have been cleaner to have Hugo handle the extra quotes for strings?

So a `Defines` here like:
```
map[string]interface{}{"process.env.NODE_ENV": "production", "integer.value": 10, "boolean.value": true}
```
Would be translated below as:
```
map[string]string{"process.env.NODE_ENV": "\"production\"", "integer.value": "10", "boolean.value": "true"}
```

I guess esbuild works like that because it's a CLI-first tool and there are no types in shells, but Hugo has the full type information.

Reply:
It's not too late, but I don't see how this would work. We know that it's a string, but we don't know that it needs to be quoted on the Esbuild side of the fence. It may resolve to a variable/constant.

Reply:
You're right. I was only thinking about literals and not that it could be referring to some variable.